Source/Core/Gen4/Searchers/WildSearcher4.cpp

```diff
@@ -366,6 +366,12 @@ std::vector<WildState> WildSearcher4::normalMethodJ(WildState state, u32 seed) c
     std::vector<WildState> states;
     state.setLead(Lead::None);
 
+    PokeRNG rng2(seed);
+    rng2.advance(4);
+    u8 heldItem = rng2.nextUShort() % 100;
+
+    state.setHeldItem(heldItem);
+
```

Comment:
Move to parent scope. Basically line 159 right before the if/else.

Will reduce code duplication and the amount of times this gets called.

Reply:
Done